src/unitxt/metrics.py

```diff
@@ -1390,9 +1386,7 @@ def verify(self):
 class MetricPipeline(MultiStreamOperator, Metric):
     main_score: str = None
     preprocess_steps: Optional[List[StreamingOperator]] = field(default_factory=list)
-    postpreprocess_steps: Optional[List[StreamingOperator]] = field(
-        default_factory=list
-    )
+    postprocess_steps: Optional[List[StreamingOperator]] = field(default_factory=list)
```

Comment:
```suggestion
    postpreprocess_steps: Optional[List[StreamingOperator]] = field(default_factory=list)
```
Non backward compatible should be in a separate pr